### PR TITLE
api retruns the new cluster resource

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**: api returns the new cluster resource

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
continuation of https://github.com/kubermatic/kubermatic/pull/1435

File: api/pkg/handler/cluster.go [hunk 1]
```diff
@@ -336,6 +349,34 @@ func listSSHKeysAssingedToCluster(sshKeyProvider provider.NewSSHKeyProvider, pro
...
+					}
+					return nil
+				}(),
+			},
+			Spec: apiv1.NewClusterSpec{
+				Cloud:   *cluster.Spec.Cloud,
+				Version: cluster.Spec.Version,
+			},
+			Status: apiv1.NewClusterStatus{
+				Version: "TODO: fill me",
```
Comment (p0lyn0mial) [inline api/pkg/handler/cluster.go]: can we just rewrite the version from `cluster.Spec.Version` ?
I think that in the future this value must be updated by the controller after the new version will be deployed.

**Review by mrIncompetent — APPROVED** [review]

File: api/pkg/handler/cluster.go (hunk 1 above)
Comment (p0lyn0mial) [reply api/pkg/handler/cluster.go]: we have to fill the `version` field,  rewriting from `cluster.Spec.Version` is okay ?

Comment (mrIncompetent) [reply api/pkg/handler/cluster.go]: sgtm

**Comment by p0lyn0mial** [comment]
I'm merging this PR into https://github.com/kubermatic/kubermatic/pull/1435

**Pull request merged**